src/huggingface_hub/keras_mixin.py

```diff
@@ -278,8 +297,8 @@ def push_to_hub_keras(
     git_email: Optional[str] = None,
     config: Optional[dict] = None,
     include_optimizer: Optional[bool] = False,
-    task_name: Optional[str] = None,
     plot_model: Optional[bool] = True,
+    tags: Optional[Union[list, str]] = None,
```

Comment:
nit: you can keep `tags` before `plot_model` so we have backwards compatibility (in same place as `task_name` was). That means that if someone (unlikely) passes all arguments, this would still work with same code as before.

Reply:
I changed this